script/prepare-comp-match

```diff
@@ -14,8 +14,7 @@ REPO_ROOT = Path(__file__).resolve().parent.parent
 sys.path.insert(1, str(REPO_ROOT / 'modules'))
 
 import controller_utils  # isort:skip
-
-NUM_ZONES = 2
+from controller_utils import NUM_ZONES  # isort:skip
```

Comment:
This seems unneessary, can you not just replace the uses of `NUM_ZONES` with `controller_utils.NUM_ZONES`?

Reply:
For sure we could, this felt like it left that code slightly clearer though (and avoided some slightly odd wrapping).

Reply:
0dfc03781cc75adb159aab5bceb513c618ce86ac